src/api/generate-xlsx/map-application-to-XLSX/map-buyer/map-previous-cover-with-buyer/index.test.ts

```diff
@@ -47,7 +49,11 @@ describe('api/generate-xlsx/map-application-to-xlsx/map-buyer/map-previous-cover
           exporterHasPreviousCreditInsuranceWithBuyer: false,
         };
 
-        const result = mapPreviousCoverWithBuyer(eligibility, mockRelationship);
+        const application = mockApplicationSinglePolicyTotalContractValueOverThreshold;
+
+        application.buyer.relationship = mockRelationship;
```

Comment:
could this and the line above go in a beforeEach?

Reply:
We can only move the `application` const above to be DRY. The `relationship` is specific to each test and has to be updated in the applicaiton after relationship stubbing in each test.